hazelcast/src/main/java/com/hazelcast/util/scheduler/EntryTaskScheduler.java

```diff
@@ -47,5 +48,14 @@
 
     void cancelAll();
 
+    /**
```

Comment:
Perhaps it is better not to expose this. I guess you need this information to determine if there are pending tasks. But there are ways to do this without needed to expose a new function on a support structure.

For example when the partition service executes a task. You could do something like this:

```
class PartitionServiceImpl ...

	final AtomicInteger pending = new AtomicInteger();
	final ScheduledExecutor executor = ..

	void dostuff(){
		Runnable task = new Runnable(){
			void run(){
                                ... do stuff
				pending.decrement();			
			}		
		}
	
		pending.increment();
		executor.submit(task, ....);
	}

}
```

As you can see you don't need to rely on exposing more functionality on the scheduler. The task can keep track of number of pending invocations.